### PR TITLE
supported --port and --host without =

### PR DESCRIPTION
**Pull request by kzidane**
Small tweak to support `--host x.x.x.x` and `--port xxxx` too.

### CODE REVIEW

File: files/home/ubuntu/.cs50/bin/flask [hunk 1]
```diff
@@ -24,12 +24,12 @@ if [[ "$1" == "run" ]]; then
...
         if [[ "$1" =~ ^--host= ]]; then
             host="$1"
-        elif [[ "$1" == "-h" ]]; then
+        elif [[ "$1" =~ ^--?h ]]; then
             host="$1 $2"
             shift
         elif [[ "$1" =~ ^--port= ]]; then
             port="$1"
-        elif [[ "$1" == "-p" ]]; then
+        elif [[ "$1" =~ ^--?p ]]; then
```
Comment (dmalan) [inline files/home/ubuntu/.cs50/bin/flask]: This would seem to match `-pfoo` and other such invalid flags?

Comment (kzidane) [reply files/home/ubuntu/.cs50/bin/flask]: Technically yes, but the current interface doesn't have `-hfoo` or `-pfoo` so we should be good?

```
Options:
  -h, --host TEXT                 The interface to bind to.
  -p, --port INTEGER              The port to bind to.
  --reload / --no-reload          Enable or disable the reloader.  By default
                                  the reloader is active if debug is enabled.
  --debugger / --no-debugger      Enable or disable the debugger.  By default
                                  the debugger is active if debug is enabled.
  --eager-loading / --lazy-loader
                                  Enable or disable eager loading.  By default
                                  eager loading is enabled if the reloader is
                                  disabled.
  --with-threads / --without-threads
                                  Enable or disable multithreading.
  --help                          Show this message and exit.
```

Comment (dmalan) [reply files/home/ubuntu/.cs50/bin/flask]: Sloppy. :p We shouldn't accept `--pfoo` and such.

File: files/home/ubuntu/.cs50/bin/flask [hunk 2]
```diff
@@ -22,14 +22,14 @@ if [[ "$1" == "run" ]]; then
...
     do
-        if [[ "$1" =~ ^--host= ]]; then
+        if [[ "$1" =~ ^--host= || "$1" =~ ^-h[^\s]+ ]]; then
             host="$1"
-        elif [[ "$1" =~ ^--?h ]]; then
+        elif [[ "$1" == "-h" || "$1" == "--host" ]]; then
             host="$1 $2"
             shift
-        elif [[ "$1" =~ ^--port= ]]; then
+        elif [[ "$1" =~ ^--port= || "$1" =~ ^-p[^\s]+ ]]; then
```
Comment (dmalan) [inline files/home/ubuntu/.cs50/bin/flask]: TIL `flask` supports this

**Pull request merged**